cmd/nginx-ingress/main_test.go

```diff
@@ -7,15 +7,15 @@ import (
 )
 
 func TestValidatePort(t *testing.T) {
-	badPorts := []int{80, 443, 1, 1022, 65536}
```

Comment:
did this one really need to change?

Reply:
No, just wanted it to check the first invalid port